api/validation/api_test.go

```diff
@@ -0,0 +1,218 @@
+package validation
+
+import (
+	"testing"
+
+	kc "github.com/cloudtrust/keycloak-client"
+	"github.com/stretchr/testify/assert"
+)
+
+func createValidUser() UserRepresentation {
+	var (
+		bFalse          = false
+		username        = "46791834"
+		gender          = "M"
+		firstName       = "Marc"
+		lastName        = "El-Bichoun"
```

Comment:
Marc El-Bichoun ? You have the same cinematographic references than mine? ^^

Reply:
I should have change the lastname, now my copy paste is obvious ;)